week1/my-app/src/CustomerService.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import deliveryLogo from "./assets/exercises/f-delivery.png"
+import coinLogo from "./assets/exercises/coin.png"
+import chatLogo from "./assets/exercises/chat.png"
+
+
+
+
+const content = [
+    { src: deliveryLogo, title: 'Free Shipping', description: 'Ta stelnoume me gaidouri' },
+    { src: coinLogo, title: '100% Money Back', description: 'Xrwstame kai ths Mixalous' },
+    { src: chatLogo, title: 'Online Support 24/7', description: 'Tha sas kanoume anapanthth na mhn xrewthoume' },
+
+];
+
+
+function Guarantee(props) {
+    const [services, setService] = useState(content)
+
+    return (
+        <div className="services">{services.map((item, index) => {
```

Comment:
Indentation. Place the children of an element in a new line
```suggestion
        <div className="services">
          {services.map((item, index) => {
```